src/components/Footer.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+const Footer = () => {
+  return (<>
+    <p>This is the footer</p>
+    </>
+  )
+}
```

Comment:
I will approve but please make sure the opening fragment tag is on its own line in all your files. 
return (
     <>